tool/elk/main.tf

```diff
@@ -0,0 +1,97 @@
+terraform {
+  backend "gcs" {
+    bucket  = "vaticle-factory-prod-terraform-state"
+    prefix  = "terraform/elk"
+  }
+}
+
+provider "google" {
+  project = "vaticle-factory-prod"
+  region  = "europe-west2"
+  zone    = "europe-west2-b"
+}
+
+resource "google_compute_firewall" "elk_api_firewall" {
+  name    = "elk-api-firewall"
+  network = "default"
+
+  allow {
+    protocol = "tcp"
+    ports    = ["443", "2053"]
```

Comment:
443: Kibana listens at this port; therefore it's accessible at https://logging.vaticle.com/
2053: ElasticSearch listens at this port; the port was chosed as this is one of the ports [that CloudFlare supports](https://support.cloudflare.com/hc/en-us/articles/200169156-Identifying-network-ports-compatible-with-Cloudflare-s-proxy) to have secure connections; it's accessible under https://logging.vaticle.com:2053